### PR TITLE
Feature/profile templates redesign

### PR DESCRIPTION
**Pull request by andriiwp3**
Made a new design for profile templates, rewrote styles, changed structure. And also added a new media request to create hover effects

### CODE REVIEW

File: src/styles/media-queries.module.scss [hunk 1]
```diff
@@ -30,6 +30,13 @@
   }
 }
 
+/* Withoute touchscreen > 767 (for hover effects) */
+@mixin without-touch-screen {
```
Comment (AndriyTykhan) [inline src/styles/media-queries.module.scss:34]: I believe we don't need this media query. We have <767 and <979. Please try to use them

File: src/components/profiles/templates/profile-book/profile-book.component.js [hunk 2]
```diff
@@ -25,62 +25,68 @@ const ProfileBook = ({ profileData }) => {
...
+                  key={file.key}
+                  className={'video-data__wrapper'}
+                />
+              ))}
+            </div>
+          </div>
+        </section>
+      ) : (
+        ''
+      )}
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-book/profile-book.component.js:89]:
```suggestion
      {otherFiles.length && (
        <section className="profile-book__page-container">
          <div className="profile-book__page">
            <h2 className="profile__title profile-book__title">Відео</h2>
            <div className="video-data__container">
              {otherFiles.map((file) => (
                <Video
                  url={getProfileDataFromBucket(file.key)}
                  key={file.key}
                  className={'video-data__wrapper'}
                />
              ))}
            </div>
          </div>
        </section>
      )}
```

File: src/components/profiles/templates/profile-simple/profile-simple.component.js [hunk 3]
```diff
@@ -22,57 +21,64 @@ const ProfileSimple = ({ profileData }) => {
...
+      <section className="profile-simple__data-container profile-simple__data-container--description-block">
+        <h1 className="title profile__name profile-simple__name">{getFullName(profileData)}</h1>
+        <p className="profile__description profile-simple__description">{description}</p>
+        {descriptionAdditional && (
+          <p className="profile__description profile-simple__description">
+            {descriptionAdditional}
+          </p>
+        )}
+      </section>
+      {otherPhotos.length ? (
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-simple/profile-simple.component.js:50]:
```suggestion
      {otherPhotos.length && (
```

File: src/components/profiles/templates/profile-simple/profile-simple.component.js [hunk 4]
```diff
@@ -22,57 +21,64 @@ const ProfileSimple = ({ profileData }) => {
...
+                img={getProfileDataFromBucket(img.key)}
+                alt="profile image"
+                className={'image-data__item image-data__item--simple'}
+              />
+            ))}
+          </div>
+        </section>
+      ) : (
+        ''
       )}
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-simple/profile-simple.component.js:66]:
```suggestion
      )}
```

File: src/components/profiles/templates/profile-simple/profile-simple.component.js [hunk 5]
```diff
@@ -22,57 +21,64 @@ const ProfileSimple = ({ profileData }) => {
...
+                url={getProfileDataFromBucket(file.key)}
+                key={file.key}
+                className={'video-data__wrapper profile-simple__video'}
+              />
+            ))}
+          </div>
+        </section>
+      ) : (
+        ''
       )}
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-simple/profile-simple.component.js:82]:
```suggestion
      )}
```

File: src/components/profiles/templates/profile-simple/profile-simple.component.js [hunk 6]
```diff
@@ -22,57 +21,64 @@ const ProfileSimple = ({ profileData }) => {
...
-      {descriptionAdditional && (
-        <p className="profile__description profile-simple__description">
-          {descriptionAdditional}
-        </p>
+      {otherFiles.length ? (
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-simple/profile-simple.component.js:67]:
```suggestion
      {otherFiles.length && (
```

File: src/components/profiles/templates/profile-article/profile-article.component.js [hunk 7]
```diff
@@ -26,65 +24,69 @@ const ProfileArticle = ({ profileData }) => {
...
+        </div>
+      </section>
+      {descriptionAdditional && (
+        <section className="profile-article__data-container">
+          <p className="profile__description profile-article__description">
+            {descriptionAdditional}
+          </p>
+        </section>
+      )}
+      {otherPhotos.length ? (
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-article/profile-article.component.js:54]:
```suggestion
      {otherPhotos.length && (
```

File: src/components/profiles/templates/profile-article/profile-article.component.js [hunk 8]
```diff
@@ -26,65 +24,69 @@ const ProfileArticle = ({ profileData }) => {
...
-      </div>
-      {descriptionAdditional && (
-        <p className="profile__description profile-article__additionalDescription">
-          {descriptionAdditional}
-        </p>
+          </div>
+        </section>
+      ) : (
+        ''
       )}
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-article/profile-article.component.js:72]:
```suggestion
      )}
```

File: src/components/profiles/templates/profile-article/profile-article.component.js [hunk 9]
```diff
@@ -26,65 +24,69 @@ const ProfileArticle = ({ profileData }) => {
...
-        <div className="video-data__container--article">
-          {otherFiles.map((file) => (
-            <Video
-              url={getProfileDataFromBucket(file.key)}
-              key={file.key}
-              className={"video-img__wrapper--article"}
-            />
-          ))}
-        </div>
+      {otherFiles.length ? (
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-article/profile-article.component.js:73]:
```suggestion
      {otherFiles.length && (
```

File: src/components/profiles/templates/profile-article/profile-article.component.js [hunk 10]
```diff
@@ -26,65 +24,69 @@ const ProfileArticle = ({ profileData }) => {
...
+                  url={getProfileDataFromBucket(file.key)}
+                  key={file.key}
+                  className={'video-data__wrapper profile-article__video'}
+                />
+              ))}
+            </div>
+          </div>
+        </section>
+      ) : (
+        ''
```
Comment (AndriyTykhan) [inline src/components/profiles/templates/profile-article/profile-article.component.js:89]:
```suggestion
      )
```

File: src/styles/media-queries.module.scss (hunk 1 above)
Comment (andriiwp3) [reply src/styles/media-queries.module.scss:34]: Yeah, we have, but we must to make a hover effects on device without touch screen, so more usable practice is to make media query > 767px (no phone and tablets, only pc and laptops). Honestly, I don`t know, how to do the same with the queries you suggested

**Comment by andriiwp3** [comment]
Okay, I`ll fix it during the day

**Pull request merged**